README.md

```diff
@@ -546,18 +546,17 @@ The primary challenges here are in achieving agreement about built-in modules on
 
 Several members of the community have been working on polyfills and tooling related to import maps. Here are the ones we know about:
 
-* [@import-maps/generate](https://www.npmjs.com/package/@import-maps/generate) generates an import map from your `yarn.lock`.
 * [@import-maps/resolve](https://www.npmjs.com/package/@import-maps/resolve) resolves a specifier relative to an import map.
 * [@jsenv/node-module-import-map](https://www.npmjs.com/package/@jsenv/node-module-import-map) generates an import map from your `package.json` and `node_modules/` directories.
-* [Snowpack](https://www.snowpack.dev) generates an import map when it installs your npm packages to run on the web.
-* [Built-in Module Demo (with Rollup)](https://glitch.com/edit/#!/rollup-built-in-modules) contains code for a [Rollup](https://rollupjs.org/) plugin that generates an import map, focused on built-in module polyfills.
+* [@jsenv/importmap-eslint-resolver](https://www.npmjs.com/package/@jsenv/importmap-eslint-resolver) enables import map resolution in [ESLint](https://eslint.org/)
```

Comment:
Good for me.

For the record I have updated @jsenv/importmap-eslint-resolver readme to make it better, thanks for the link ;)